specs/analytics_v1_pageviews.yaml

```diff
@@ -0,0 +1,291 @@
+  contact:
+    name: Analytics
+    email: analytics@lists.wikimedia.org
+    url: https://www.mediawiki.org/wiki/Analytics
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+x-default-params:
+  domain: analytics.wikimedia.org
+paths:
+  /{module:pageviews}/:
+    get:
+      tags:
+        - Pageviews data
+      description: >
+        List pageviews data sub-apis.
+
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Stable)
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: The queriable sub-items
+          schema:
+            $ref: '#/definitions/listing'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-monitor: false
+
+  /{module:pageviews}/per-article/{project}/{access}/{agent}/{article}/{granularity}/{start}/{end}:
+    get:
+      tags:
+        - Pageviews data
+        - per-article
+      description: >
+        List pageviews for a given article in a project, over a given time period.
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: project
+          in: path
+          description: Name of the project
+          type: string
+          required: true
+        - name: access
+          in: path
+          description: Type of access method [all-access|desktop|mobile-app|mobile-web]
+          type: string
+          required: true
+        - name: agent
+          in: path
+          description: Type of user agent [all-agents|user|spider]
+          type: string
+          required: true
+        - name: article
+          in: path
+          description: Name of the article for which to get data
+          type: string
+          required: true
+        - name: granularity
+          in: path
+          description: Time granularity of the response data [hourly|daily]
+          type: string
+          required: true
+        - name: start
+          in: path
+          description: Start timestamp in YYYYMMDDHH format
+          type: string
+          required: true
+        - name: end
+          in: path
+          description: End timestamp in YYYYMMDDHH format
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The list of values
+          schema:
+            $ref: '#/definitions/article'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-backend-request:
+        uri: /{domain}/sys/pageviews/per-article/{project}/{access}/{agent}/{article}/{granularity}/{start}/{end}
+      x-monitor: false
+
+  /{module:pageviews}/per-project/{project}/{access}/{agent}/{granularity}/{start}/{end}:
+    get:
+      tags:
+        - Pageviews data
+        - per-project
```

Comment:
I'd avoid too many tags, because in [swagger-driven docs](https://rest.wikimedia.org/en.wikipedia.org/v1/?doc) each tag becomes a separate section, so it's better to keep all page views endpoints under a single tag.